src/backend/distributed/commands/utility_hook.c

```diff
@@ -1044,16 +1044,20 @@ ExecuteDistributedDDLJob(DDLJob *ddlJob)
 
 	EnsureCoordinator();
 
-	Oid targetRelationId = ddlJob->targetRelationId;
+	ObjectAddress targetObjectAddress = ddlJob->targetObjectAddress;
 
-	if (OidIsValid(targetRelationId))
+	if (OidIsValid(targetObjectAddress.classId))
```

Comment:
Should we maybe use `targetObjectAddress.classId == RelationRelationId`  because it is not an oid, and probably we'll soon have other classes?

Reply:
I'm planning to handle other objects as a part of `ShouldSyncUserCommandForObject`, so keeping it as it is.